### ISSUE TITLE
(REQUEST) Add 25 fps on ps vita among the selectable caps.

### ISSUE THREAD

**Issue by Vinnievovi** [Electry/VitaGrafix#218]
(REQUEST) Add 25 fps on ps vita among the selectable caps, especially for Jak and Daxter.

**Comment by pixceta** [comment]
i dont think that is possible, i am no expert but i think that would lead to screen tearing without v-sync, which would make the game unplayable
